nss-day-cohort-47/reactive-nutshell-palm-beach-pilots
# Articles getfunctionality

# Description
Adds the initial functionality of the Articles component. The component lists articles by user and by the users friends. 
Fixes # (issue)
## Type of change
Please delete options that are not relevant.
# Testing Instructions
This pull request can be tested by opening the app and clicking on articles. It should show the articles that the user has posted along with a source link. This link will open a new tab wi this site. There are also buttons for delete an edit, though they don't do anything yet.
# Checklist:
- [ -] My code follows the style guidelines of this project
- [- ] I have commented my code, particularly in hard-to-understand areas
- [- ] My changes generate no new warnings or errors
- [ -] I have added test instructions that prove my fix is effective or that my feature works
